### PR TITLE
Decode PKCS12 keystore with multiple keypairs

### PR DESCRIPTION
**Pull request by godrei**
This PR updates keystore decoding to support PKCS12 keystores with multiple keypairs in it.

[KeyStore Explorer tool](https://github.com/kaikramer/keystore-explorer) was used to generate the test keystore (`testdata/multi_keys.pkcs12`).

### CODE REVIEW

File: pkcs12.go [hunk 1]
```diff
@@ -418,30 +419,26 @@ func DecodeKeystore(pfxData []byte, storepass, alias, keypass string) (privateKe
...
 		case bag.Id.Equal(oidCertBag):
-			certAlias, err = getAlias(bag)
+			certAlias, err := getAlias(bag)
 			if err != nil {
 				return nil, nil, fmt.Errorf("pkcs12: failed to get alias for the cert bag: %s", err)
 			}
 
+			if strings.ToLower(certAlias) != strings.ToLower(alias) {
+				continue
+			}
```
Comment (tothszabi) [inline pkcs12.go:440]: Does this mean that the old implementation might not worked properly? In the old implementation we actually did not user the alias in this method.

**Review by tothszabi — APPROVED** [review]

File: pkcs12.go (hunk 1 above)
Comment (godrei) [reply pkcs12.go:440]: I don't think so, at the end of the function we checked if the certAlias = privateKeyAlias: https://github.com/bitrise-io/go-pkcs12/pull/4/files#diff-93d044ade6a726f2b3e0ce7975e65d8db5f639465c90b4d0f3aa159871f309caL479

Comment (tothszabi) [reply pkcs12.go:440]: Sorry, my bad. 👍🏻

**Comment by lpusok** [comment]
Is it not DecodeAll doing this already? https://github.com/bitrise-io/go-pkcs12/blob/51c27da56110ab44e2f20e5416978ce0f8d2e898/pkcs12.go#L324

**Pull request merged**